client/me/notification-settings/settings-form/device-selector.jsx

```diff
@@ -32,12 +31,18 @@ export default React.createClass( {
 					<FormSelect
 						value={ this.props.selectedDeviceIndex }
 						onChange={ this.props.onChange } >
-						{ this.props.devices.get().map( ( device, index ) => {
-							return <option key={ index } value={ index }>{ device.device_name }</option>;
+						{ map( devices, ( device, index ) => {
+							return <option key={ index } value={ index }>{ device.name }</option>;
```

Comment:
could we use the device name as the key here? we want to be very careful with using indices as keys because we can introduce bugs if the list order happens to change.